tests/drivers/build_all/led/app.overlay

```diff
@@ -31,5 +31,45 @@
 				label = "Test GPIO LED";
 			};
 		};
+
+		test_i2c: i2c@11112222 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			compatible = "vnd,i2c";
+			reg = <0x11112222 0x1000>;
+			label = "TEST_I2C_CTLR";
+			status = "okay";
+			clock-frequency = <100000>;
+
+			test_i2c_lp5009: lp5009@14 {
```

Comment:
I would move it to the separate `i2c.dtsi` like here https://github.com/zephyrproject-rtos/zephyr/blob/6b1d7aa08ba82d03a2de63d02d443124ec2f1b1c/tests/drivers/build_all/sensor/app.overlay#L38=. I know that there are only 2 nodes right now but it will be helpful for the future.